contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -264,6 +271,23 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, IERC677Rece
     return s_subscriptions[subscriptionId];
   }
 
+  // @inheritdoc IFunctionsSubscriptions
+  // function getSubscriptionsInRange(
```

Comment:
Maybe sacrifice getSubscription() and only use this new one to get some gas back?

Reply:
Yeah that makes sense, I think that'll get us there

Reply:
Didn't end up needing to do this